### PR TITLE
Fix bug #1293

### LINKED ISSUE #1293: core/vendor/.gitignore ignores too much
core/vendor/.gitignore has these lines:

jquery/**
!jquery/dist/jquery.*

the second line has no effect because the parent directory of dist is ignored (see the man page for .gitignore). The same thing with

clipboard/**
!clipboard/dist/clipboard.js

Also the lines

.gitignore
davclient.js/**

cause files to be ignored

### Steps to reproduce
1. git clone https://github.com/nextcloud/server
2. cd nextcloud
3. git rm -r --cached core/vendor
4. git add core/vendor
5. git status --ignored

### Expected behaviour
All files should get added back to the index

### Actual behaviour
the output shows:

```

Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        deleted:    core/vendor/.gitignore
        deleted:    core/vendor/clipboard/dist/clipboard.js
        deleted:    core/vendor/davclient.js/lib/client.js
        deleted:    core/vendor/jquery/dist/jquery.js
        deleted:    core/vendor/jquery/dist/jquery.min.js
        deleted:    core/vendor/jquery/dist/jquery.min.map

Ignored files:
  (use "git add -f <file>..." to include in what will be committed)

        core/vendor/.gitignore
        core/vendor/clipboard/
        core/vendor/davclient.js/lib/
        core/vendor/jquery/dist/
```

If you copy the nextcloud installation package to a fresh git repository, those files will stay out of the index and nextcloud won't work.

### PR DESCRIPTION
**Pull request by tverho**
Fix #1293

### CODE REVIEW

File: core/vendor/.gitignore [hunk 1]
```diff
@@ -126,6 +126,8 @@ backbone/backbone-min*
 # davclient.js
 davclient.js/**
 !davclient.js/lib
+!davclient.js/lib/client.js
+!davclient.js/lib
```
Comment (nickvergessen) [inline core/vendor/.gitignore]: line already exists 2 lines above?

**Comment by MorrisJobke** [comment]
Makes sense 👍

**Comment by MorrisJobke** [comment]
cc @rullzer

**Comment by rullzer** [comment]
LGTM

**Comment by rullzer** [comment]
Thanks a lot @tverho !

**Comment by tverho** [comment]
Thanks for quick processing!

**Pull request merged**